cross-modal-search/pods/executors.py

```diff
@@ -8,7 +8,7 @@
 from PIL import Image
 import io
 from jina import Executor, DocumentArray, requests, Document
-from jina.helloworld.chatbot.my_executors import _norm, _ext_B, _ext_A, _cosine
+from jina.helloworld.multimodal.my_executors import _norm, _ext_B, _ext_A, _cosine
```

Comment:
copy the code of `_norm, _ext_B, _ext_A, _cosine`

Reply:
Addressed, please see [here](https://github.com/jina-ai/examples/pull/684/commits/b941d959c228d88758aff43dfcf2a7004965f2db) for the change.